src/visitor/mysql.rs

```diff
@@ -12,6 +12,7 @@ use std::fmt::{self, Write};
 pub struct Mysql<'a> {
     query: String,
     parameters: Vec<Value<'a>>,
+    table: Option<Table<'a>>,
```

Comment:
Would you mind either being a bit more specific on the name of the field or just adding docs that explain what this is used for? `table` is pretty generic in the context of the visitor given that it's only set when visiting an update or a delete stmt.